code/modules/client/preferences_savefile.dm

```diff
@@ -5,7 +5,7 @@
 //	You do not need to raise this if you are adding new values that have sane defaults.
 //	Only raise this value when changing the meaning/format/name/layout of an existing value
 //	where you would want the updater procs below to run
-#define SAVEFILE_VERSION_MAX	33
+#define SAVEFILE_VERSION_MAX	34 // yogs edit -- bump savefile verison
```

Comment:
Do you need to add a new version if its an addition?

Reply:
Not sure - this just enables VOX by default in all old savefiles